app/src/main/res/layout/activity_create_activity.xml

```diff
@@ -0,0 +1,278 @@
+                                android:layout_marginHorizontal="10dp"
+                                android:background="@color/white"
+                                android:ems="10"
+                                android:inputType="time"
+                                android:hint=" ex: 19h00"
+                                android:textColorHint="#DEE5E9"
+                                android:textColor="@color/black" />
+                            <EditText
+                                android:id="@+id/event_end_time"
+                                android:layout_width="160dp"
+                                android:layout_height="30dp"
+                                android:layout_marginHorizontal="10dp"
+                                android:background="@color/white"
+                                android:ems="10"
+                                android:inputType="time"
+                                android:hint=" ex: 23h00"
+                                android:textColorHint="#DEE5E9"
+                                android:textColor="@color/black" />
+                        </LinearLayout>
+
+                        <TextView
+                            android:id="@+id/textView9"
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:layout_marginHorizontal="10dp"
+                            android:layout_marginTop="10dp"
+                            android:textColor="@color/black"
+                            android:textSize="20dp"
+                            android:text="Event Description:" />
+                        <EditText
+                            android:id="@+id/event_description"
+                            android:layout_width="match_parent"
+                            android:layout_height="100dp"
+                            android:layout_marginHorizontal="10dp"
+                            android:background="@color/white"
+                            android:ems="10"
+                            android:inputType="textMultiLine"
+                            android:hint=" Description..."
+                            android:textColorHint="#DEE5E9"
+                            android:textColor="@color/black"
+                            android:gravity="top" />
+
+                        <TextView
+                            android:id="@+id/textView10"
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:layout_marginHorizontal="10dp"
+                            android:layout_marginTop="10dp"
+                            android:textColor="@color/black"
+                            android:textSize="20dp"
+                            android:text="Maximum number of participants:" />
+                        <EditText
```

Comment:
I think the EditTexts can have a more rounded shape to match the UI more closely. I already made such a shape so you can simply add this line: `android:background="@drawable/edit_text_shape"`